x-pack/plugins/security_solution/public/timelines/containers/helpers.ts

```diff
@@ -4,14 +4,32 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { isEqual } from 'lodash/fp';
```

Comment:
nit: Favor module import to save on size -- this change is coming globally as part of https://github.com/elastic/kibana/pull/74539
```suggestion
import isEmpty from 'lodash/isEmpty';
```

Reply:
Good point, will make this minor change.

Reply:
Well, looking through all the code it looks like this is the pattern we use which is destructoring it, so I would keep the 
same pattern here for now. If we want to refactor through a code sweep we can and I think that would be better:

Pattern search of isEqual:
<img width="774" alt="Screen Shot 2020-08-24 at 4 15 21 PM" src="https://user-images.githubusercontent.com/1151048/91101822-2197db00-e625-11ea-8a05-5477c14877f2.png">

Pattern search of all lodash
<img width="768" alt="Screen Shot 2020-08-24 at 4 14 57 PM" src="https://user-images.githubusercontent.com/1151048/91101827-25c3f880-e625-11ea-9f26-3147ab345b97.png">

Reply:
Oh wait, I see what you're saying. That upcoming PR is going to change _everything_ and I should go ahead and change it now here as well. Let me do that now.